widget_driver_generator/lib/src/code_providers/driver_provider_code_provider.dart

```diff
@@ -1,5 +1,6 @@
 import 'package:widget_driver_generator/src/utils/class_utils.dart';
 import 'package:widget_driver_generator/src/utils/code_writer.dart';
+import 'package:widget_driver_generator/src/utils/string_extensions.dart';
```

Comment:
I guess all of these generator changes you will remove from here and handle in the other PR?
Or you want to kill the other PR and handle it all in this PR?

Reply:
Handeled with the other PR and then removed or obsolete once that on is merged